com.unity.multiplayer.mlapi/Editor/NetworkAnimatorEditor.cs

```diff
@@ -1,94 +0,0 @@
-using System;
-using MLAPI.Prototyping;
-using UnityEditor.Animations;
-using UnityEngine;
-
-namespace UnityEditor
-{
-    [CustomEditor(typeof(NetworkAnimator), true)]
-    [CanEditMultipleObjects]
-    public class NetworkAnimatorEditor : Editor
```

Comment:
is there anything replacing this or do you think default inspector UI (Editor) is good enough now?